src/services/OswServiceTasks.cpp

```diff
@@ -40,8 +38,6 @@ OswServiceTask* oswServiceTasks[] = {
 #ifdef OSW_FEATURE_WIFI
     & OswServiceAllTasks::wifi, &OswServiceAllTasks::webserver,
 #endif
-#ifndef NDEBUG
     & OswServiceAllTasks::memory
```

Comment:
remove spacing

Reply:
`& OswSe..`

Reply:
Nope - but I have to admit, not the best style:
```
2256587a (GitHub Action 2022-05-31 13:04:26 +0000 31)     & OswServiceAllTasks::bleCompanion,
2256587a (GitHub Action 2022-05-31 13:04:26 +0000 39)     & OswServiceAllTasks::wifi, &OswServiceAllTasks::webserver,
```